acc.config.json

```diff
@@ -8,7 +8,7 @@
     "themeColor": "#ec0000"
   },
   "logo": {
-    "url": "/assets/logos/logo-weesp.svg",
+    "url": "/assets/logos/logo-weesp-aligned.svg",
```

Comment:
Why is it necessary to rename the file?

Reply:
**logo-weesp.svg** - Original logo provided by Weesp
**logo-weesp-aligned.svg** - Logo with alignment hack by Herjen

I thought it would be nice to keep the original in the code base since the one with `the alignment hack` 'feels' temporarily.